### DISCUSSION TITLE
which type of python version should we use

### DISCUSSION THREAD

**Discussion by MuhammadNizamani** [MuhammadNizamani/nizchain#6] [Q&A] [answered]
I am thinking about use python 3.11 @Salman-18  please discuss this.

**Comment by Salman-18** [comment, marked as answer]
Using Python 3.11 is a great idea! It offers significant performance improvements (up to 60% faster than Python 3.10), better debugging with precise error locations, and new features like exception groups and improved async handling. Most major libraries now support Python 3.11, making it a reliable and efficient choice for our projects.

**Reply by MuhammadNizamani** [reply]
So can you change the python version in linting action please

**Reply by Salman-18** [reply]
I've updated our linting action to use Python 3.11 instead of 3.10.
